src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.html

```diff
@@ -1,60 +1,57 @@
-<div class="modal-header">
-  <button type="button"
-          class="close float-right"
-          aria-label="Close"
-          (click)="modalRef.hide()">
-    <span aria-hidden="true">&times;</span>
-  </button>
-</div>
-<div class="modal-body">
-  <h2>
-    <img src="assets/Ceph_Logo_Stacked_RGB_120411_fa_348x348.png"
-         class="ceph-logo"
-         alt="{{ projectConstants.organization }}">
-    {{ projectConstants.organization }}
-  </h2>
-  <h3>
-    <strong>{{ projectConstants.projectName }}</strong>
-  </h3>
-  <div class="product-versions">
-    <strong>Version</strong>
-    <br>
-    {{ versionNumber }}
-    {{ versionHash }}
+<div class="about-container">
+  <div class="modal-header">
+    <button type="button" class="close float-right" aria-label="Close" (click)="modalRef.hide()">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="modal-body">
+
+    <img src="assets/Ceph_Logo_Standard_RGB_Reversed_120411_fa_cropped.png" class="ceph-logo"
```

Comment:
Rather than creating a new asset (cropped version) from an existing one, why not using CSS tricks to achieve that? There are at least 3 different ways to crop a picture in CSS (`clip-path`, `object-fit` and box-model).

Reply:
Thanks  I got it to work with image-fit.